calendar-amqp/src/main/java/com/linagora/calendar/amqp/EventAlarmConsumer.java

```diff
@@ -239,4 +256,10 @@ private Mono<AccountId> getAccountId(OpenPaaSId openpaasUserId) {
                 }
             }));
     }
+
+    private Mono<Boolean> getUserSettingsEnable(Username user) {
+        return settingsBasedResolver.readSavedSettings(sessionProvider.createSession(user))
+            .flatMap(settings -> Mono.justOrEmpty(settings.get(ALARM_SETTING_IDENTIFIER, Boolean.class)))
+            .defaultIfEmpty(DISABLE_ALARM);
```

Comment:
.defaultIfEmpty(ENABLE_ALARM)

(We want a setting to DISABLE it default is indeed ENABLED)